### ISSUE TITLE
Need help from wordpress/php developer for a lazySizes wordpress plugin

### ISSUE THREAD

**Issue by aFarkas** [aFarkas/lazysizes#45]
I started a [simple wordpress plugin](https://github.com/aFarkas/wp-lazysizes) to bring lazySizes to wordpress. While this plugin already works great, I'm clearly not a php developer nor a wordpress plugin author and would like to hand this project over to someone more capable doing this.

One big goal is to work seamlessly with the following responsive images plugin (and maybe others):
https://wordpress.org/support/plugin/ricg-responsive-images

In total I don't expect too much work for a php dev (should stay simple).

So in case you are one or you know someone, let me know.

**Comment by TrilipuT** [comment]
Hi!
I have used lazysizes on couple of my wp projects. I plan to get all code into plugin, but had no time for this :) I will try to contribute here )

**Comment by zacharybrady** [comment]
I've augmented this plugin https://github.com/ResponsiveImagesCG/wp-tevko-responsive-images to work with Lazysizes. Pretty easy to do. A big note is that you don't need the bit of code related to shortcodes anymore. Its better if your function just returns the pure HTML back to the editor.

**Comment by TrilipuT** [comment]
I used same logic in my code. But i have logic that creates LQIP sizes automatically.
Also i set up all settings including sizes in code, but for plugin need to place it in settings.
+ i have code to make lazyload content images.

**Comment by aFarkas** [comment]
While I have no knowledge about wordpress I think this is already too much of work. Main purpose of the wp-lazysizes plugin is to lazyload responsive images without managing the srcset/picture generation itself. It should simply hook into wordpress and filter and alter the markup so that normal images as also responsive images (pure ``srcset`` with or without ``picture`` wrapper) and iframes can be lazyloaded by lazysizes. As also providing some features as  settings or helper  functions to make working with responsive images easier (configuring some options, maybe auto adding of lqip source, generating of an intrinsic ratio wrapper).

The plugin itself should not make it possible to generate ``srcset``, this is the purpose of other plugins (especially for example https://wordpress.org/support/plugin/ricg-responsive-images). It doesn't make sense to duplicate the functionality of the ricg-responsive-images plugin. 

Basically the most important parts of the described functionality is already included, but I really want to have someone who is capable of a) reviewing it, b) maintaining it, c) releasing it and/or d) being the owner of this project, because PHP is over my capabilities.

**Comment by TrilipuT** [comment]
Yes, i understand you.
Then we need just a bit capabilities that i wrote )
I have simple lazyload img's (without LQIP), iframes lazyload. I think will be nice to have admin page where you can select which img_sizes you need to lazyload (i made this directly in code). 
If this is ok for you - i will make pull request.

And checked your code - you have already made nice work on this plugin :)

**Comment by aFarkas** [comment]
https://github.com/aFarkas/wp-lazysizes/issues/5
